tests/parser/test_parser.py

```diff
@@ -103,6 +105,74 @@ def test_parse_md_details(caplog):
     assert header.dep_labels == ["compound", "ROOT"]
 
 
+def test_parse_md_paragraphs(caplog):
```

Comment:
Add some checks for logic like some sentence's parent is some paragraph, section etc..

Reply:
Sentence can only have a Paragraph parent.